src/api/goods.ts

```diff
@@ -10,10 +10,15 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => {
+      return goods.sort((good1, good2) => good1.name.localeCompare(good2.name))
```

Comment:
You could use arrow function as a callback for `then` block. It may looks better.